src/widget/findonweb.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <QMenu>
+
+class Track;
+
+class FindOnWeb { // : public QMenu // Q_OBJECT // Interface
```

Comment:
I found something while I am looking for solving the issue with connect on 4772.

On the [QT signals and slots Doc ](https://doc.qt.io/qt-5/signalsandslots.html) It says:
> The other way to connect a signal to a slot is to use [QObject::connect](https://doc.qt.io/qt-5/qobject.html#connect)() and the SIGNAL and SLOT macros.

Can I use the QObject:: , I have only seen few examples on the codebase

Reply:
So FindOnWeb should inherit from Q_Object? or QMenu?

What I thought in the first place, a parented QMenu from WFindOnWeb and their sub classes are the services, would it be correct?

Reply:
The FindOnWeb menu is the parent class of the FindOnDiscogs, in sense of the Qt object tree. Basically the stack of widgets you see in the GUi

It is not a parent (base class) in sence of inheritance an C++
Because both use the human family naming, it is confusing.

Reply:
Nice!